language-support/java/bindings-rxjava/src/main/java/com/daml/ledger/rxjava/grpc/UserManagementClientImpl.java

```diff
@@ -36,7 +36,7 @@ public UserManagementClientImpl(
             StubHelper.authenticating(this.serviceFutureStub, maybeToken)
                 .createUser(request.toProto()),
             sequencerFactory)
-        .map(User::fromProto);
+        .map(res -> User.fromProto(res.getUser()));
```

Comment:
A wrapper type for the Protobuf should be added and that should be returned, rather than keeping the `User` type. Otherwise we lose the advantage of being able to evolve the API over time and this is especially important for the Java bindings, which are officially supported for external customers. I'm ok merging this as is and doing the work myself, just let me know what's your preference.

Reply:
Would be great if you could handle it, thank you!